dns/dnssec.py

```diff
@@ -39,27 +39,27 @@ class ValidationFailure(dns.exception.DNSException):
 
 
 #: RSAMD5
-RSAMD5 = 1
+ALGO_RSAMD5 = 1
```

Comment:
Unfortunatelly this change is backwards incompatible. It break code which e.g. looks for DNSSEC algorithm numbers in RR data or similar. Example of such code is here:
https://github.com/kirei/dnssec-ta-tools/blob/master/csr2dnskey/csr2dnskey.py#L69

Reply:
@danielrobbins can you remove ```ALGO_``` prefix please? That change seems is required by dnspythondome.